scripts/bert/dataset.py

```diff
@@ -331,6 +338,27 @@ def dataset_test(self):
         return [('test_matched', self.get_dataset(segment='test_matched')),
                 ('test_mismatched', self.get_dataset(segment='test_mismatched'))]
 
+class XNLITask(GlueTask):
+    """The XNLI task."""
```

Comment:
The XNLI task using the dataset released from Baidu <add link to Baidu's repo>

Reply:
added

Reply:
added